src/utils/dmg.c

```diff
@@ -259,6 +265,28 @@ destroy_hdlr(int argc, char *argv[])
 		return 2;
 	}
 
+	if (force) {
+		/* TODO evict can be done with destory in a single RPC
+		 * once it send destory RPC to the pool leader.
+		 */
+		if (svc_str == NULL) {
+			fprintf(stderr, "Provide service --svc=xx with -f\n");
+			return 2;
+		}
+
+		svc = daos_rank_list_parse(svc_str, ":");
+		if (svc == NULL) {
+			fprintf(stderr, "failed to parse service ranks\n");
+			return 2;
+		}
+
+		rc = daos_pool_evict(pool_uuid, sysname, svc, NULL /* ev */);
+		if (rc != 0) {
+			fprintf(stderr, "failed to evict pool: %d\n", rc);
+			return rc;
+		}
```

Comment:
dmg will be eventually removed, so we need a solution for daos_shell too.
For daos_shell, could we just add a force flag on destroy and call on the server via drpc evict and then destroy? @tanabarr any thoughts?

Reply:
Indeed, probably evict from the server side would be a better choice. But I am not an expert for those dRPC stuff. Could some one from metadata team take care of this?

Reply:
will probably need to split ds_pool_evict_handler into RPC specific and internal functions then call internal from the dRPC handler. then would be executed as you mention @johannlombardi